lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -295,6 +291,11 @@ private JourneyResponse getJourneyResponse(
                 return JOURNEY_REPROVE_IDENTITY_GPG45_MEDIUM;
             }
 
+            // PYIC-6901 Confirm that we only want to compare CI scores against the lowest requested
+            // VOT
```

Comment:
Is this comment still needed? we said new/custom mitigation journeys are out of scope for MVP

Reply:
Maybe we should get rid of the comment, but this needs to be changed too I think. On 6901 you said that if a user needs to mitigate a P2 but in the meantime comes back to us for a P1 we should continue the P2 mitigation, but I don't think that will happen if they are breaching a P2 but not a P1.

Reply:
I'm not sure how we would know what mitigation they are in the middle of

Reply:
That sounds reasonable for now, I'd say any special handling of it beyond what we already have is out of scope for MVP - but good to keep a reference I guess if we want to come back to this in future